src/components/abstractComponent.ts

```diff
@@ -226,7 +223,7 @@ export module Component {
      * @param {number} [availableHeight] - the height of the container element
      * @returns {Component} The calling component.
      */
-    public resize(width?: number, height?: number): AbstractComponent {
+    public forceRedraw(width?: number, height?: number): AbstractComponent {
```

Comment:
Agreed with @bluong here, the only minor difference I have with his proposal is requiring both width and height to be set for the rootSVG size to be set as opposed to one or the other.